lib/models/authentication/authentication_service.dart

```diff
@@ -187,11 +192,10 @@ class AuthenticationService {
 
       final UserDetail result = UserDetail(
         uid: user.uid,
-        email: email,
+        email: credential.email,
```

Comment:
we do not need this change, `email` will take value of `credential.email` if it's available, otherwise it will take value of `user?.email`